wireapp/wire-android-reloaded
# chore: video frozen state when minimized

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

[AR-2183 - During call when screen minimised, remote sees frozen screen](https://wearezeta.atlassian.net/browse/AR-2183)

### Issues

Remote User would see frozen video instead of `Video Paused`

### Causes (Optional)

Video pause was not handled when minimizing the ongoing call screen

### Solutions

1. Add `pauseVideo()` to `navigateBack()` in `SharedCallingViewModel`
2. Add hardware back press handler `BackHandler()` in `OngoingCallScreen` to call `navigateBack()

### Testing

#### How to Test
______
- [Test 1]
- Open App
- Receive/Start Call and start video
- Click down arrow in top left corner to minimize ongoing call screen
- other user in call should see Paused Video instead of frozen camera
______
- [Test 2]
- Open App
- Receive/Start Call and start video
- Press back button from device (hardware back button) to minimize ongoing call screen
- other user in call should see Paused Video instead of frozen camera
